tools/tasks/css_task.ts

```diff
@@ -1,9 +1,11 @@
 import { Task } from './task';
+import Config from '../config';
 
 export abstract class CssTask extends Task {
 
+    // Is it still usefull if cache used ?
   shallRun(files: String[]) {
-    return files.some(f =>
+    return Config.ENABLE_SCSS || files.some(f =>
```

Comment:
I see. Ok, it'll be great if you can add a comment for this. So basically the task will run if the file changed is `.css` or in all cases when we're using Sass, however, we won't have performance implications because of the caching that you introduced. Correct?